optaplanner-distribution/src/main/assembly/assembly-optaplanner.xml

```diff
@@ -64,31 +64,32 @@
       </excludes>
     </fileSet>
     <!-- Quickstart -->
-    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
-      <directory>../optaplanner-quickstarts</directory>
-      <outputDirectory>quickstarts</outputDirectory>
-      <excludes>
-        <exclude>pom.xml</exclude>
+<!--    <fileSet>&lt;!&ndash; Note: going outside the module dir is bad, but it is not fetching generated files &ndash;&gt;-->
```

Comment:
Do we need to keep the commented-out assembly together will all the exclusions?

Reply:
We decided that the zip requires the binaries of the quickstarts, but not the sources. So no, you're right. I'll remove this.